### ISSUE TITLE
One line if statement

### ISSUE THREAD

**Issue by jmcrey** [BAntonellini/tomerater#3]
https://github.com/BAntonellini/tomerater/blob/e9ed844f2ca6b747f2b89c3ce9f77c26a9917579/TomeRater.py#L58-L59

Great job with these if statements! They work great, are very explicit, and satisfy the project requirements. I just wanted to show you a trick on how to make this into one line. So, since Python has left-to-right "lazy" evaluation (i.e. will stop evaluating a boolean once it knows it's wrong), then we can make these two if statements into one, like so:

`if rating != None and rating >= 0 and rating <= 4`

Basically, Python will first check if rating is None, then it will check if it's greater than or equal to 0, then it will check if it's less or equal to than 4. If rating is None, it will stop evaluating the statement without continuing (because of lazy evaluation). Similar thing happens if it's less than 0.

That being said, the way it is done before is perfectly acceptable -- this is just a matter of preference!
